sdk/tables/Azure.Data.Tables/tests/TableServiceClientLiveTests.cs

```diff
@@ -95,5 +95,68 @@ public async Task GetTablesReturnsTablesWithFilter()
                 }
             }
         }
+
+        [Test]
+        public async Task GetPropertiesReturnsProperties()
+        {
+            // Get current properties
+            TableServiceProperties beforeResponse = await service.GetPropertiesAsync().ConfigureAwait(false);
+
+            // Create list of CorsRules
+            IList<CorsRule> corsRules = new List<CorsRule>();
+
+            // Create TableServiceProperties to set
+            var tableServicePropertiesToSet = new TableServiceProperties(
+                new LoggingSettings("1.0", !beforeResponse.Logging.Delete, true, true, new RetentionPolicy(false)), hourMetrics: new TableMetrics("1.0", true, true, new RetentionPolicy(true, 7)), minuteMetrics: new TableMetrics("1.0", false, null, new RetentionPolicy(false)), corsRules);
+
+            // Set properties
+            await service.SetPropertiesAsync(tableServicePropertiesToSet).ConfigureAwait(false);
+
+            // Wait 20 seconds if on Live mode
```

Comment:
could we add a comment here on the why? Is it a recommended waiting time?
I wonder if there is another way to see if the properties are propagated and ready to consume.

Reply:
It was around the minimum amount of time for the property changes to be made for the sync methods.